src/App.tsx

```diff
@@ -56,6 +58,30 @@ const App = () => (
               </BaseLayout>
             )}
           />
+          <Route
+            path="/page-not-found"
```

Comment:
During this sprint, @dlrice defined routes in as an enum (see all routes above). I think we should be consistent and use this system for all routes.

Reply:
I don't if that was there when I initially added my routes, but I have now moved all of them to the `Location` enum.